src/Dfe.PlanTech.Web/Views/Shared/_CookieBanner.cshtml

```diff
@@ -35,15 +31,22 @@
       </div>
     </form>
 } 
-else if (accepted && !hidden)
+else if ((cookie.HasApproved || cookie.IsRejected) && cookie.IsVisible)
 {
     <form method="post" asp-controller="Cookies" asp-action="HideBanner">
         <div class="govuk-cookie-banner " data-nosnippet role="region" aria-label="Cookies on Plan technology for your school">
             <div class="govuk-cookie-banner__message govuk-width-container">
                 <div class="govuk-grid-row">
                     <div class="govuk-grid-column-two-thirds">
                         <div class="govuk-cookie-banner__content">
-                            <p class="govuk-body">You’ve accepted additional cookies. You can <a class="govuk-link" href="#">change your cookie settings</a> at any time.</p>
+                            @if (cookie.HasApproved && cookie.IsVisible) 
+                            {
+                                <p class="govuk-body">You’ve accepted additional cookies. You can <a class="govuk-link" href="#">change your cookie settings</a> at any time.</p>
```

Comment:
Can we change the href to be `/cookies` please?

Reply:
Added